src/asciidoc/docs/develop/publishLocal.adoc

```diff
@@ -25,28 +27,14 @@ See the first part of the tutorial.
 * Install Node.js and launch your project locally.
 See the second part of the tutorial.
 
-== Changing a custom value
+== Making a change in the source code
 
-For the https://withkoji.com/~Svarog1389/element-match[Element Match] game, a new game title would be a simple change that is obvious to test.
-To change the title in the Koji editor, you would click *VCC Explorer > Game Settings* on the left side, which opens the Game Settings editor in Visual mode.
-This mode shows the visual controls that will enable non-technical users to customize your template when you have finished developing it.
+Let's make a small change in the https://withkoji.com/~Svarog1389/rxkd[Falling Objects] game, just for testing purposes.
 
-Because Visual mode is not available when you are developing your project locally, you must edit the JSON customization file directly.
+. Open `frontend\src\Components\View\LeaderboardMini.js`.
+Change `<Title>{`Top Players`}</Title>` to `<Title>{`Leaderboard`}</Title>` and save the file.
```

Comment:
The formatting is a bit broken. I think the backticks in the code sample are breaking the parsing. You might need to change the character or escape them.